packages/modals/src/containers/FocusJailContainer.spec.js

```diff
@@ -130,11 +128,11 @@ describe('FocusJailContainer', () => {
                 <button>Another button</button>
               </div>
             )}
-          </FocusJailContainer>,
-          { enzymeOptions: { attachTo: document.body } }
+          </FocusJailContainer>
         );
+
         focusElementSpy.mockClear();
-        wrapper.simulate('keydown', { keyCode: KEY_CODES.TAB });
+        fireEvent.keyDown(container.firstChild, { keyCode: KEY_CODES.TAB });
```

Comment:
You could do `getByTestId('container')` to more explicit about which item the event is firing on.